sdk/confidentialledger/confidential-ledger-rest/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/confidential-ledger"
+      ServiceDirectory: confidentialledger
+      EnvVars:
+        LEDGERURI: https://$(ledgerName).confidential-ledger.azure.com
+        LEDGER_IDENTITY: $(ledgerName)
+        IDENTITY_SERVICE_URL: https://identity.confidential-ledger.core.azure.com/ledgerIdentity/$(ledgerName)
+        LEDGER_NAME: $(ledgerName)
```

Comment:
These should be set as ARM template output variables instead of being hardcoded in the yaml, to keep the configs tidy and avoid us having to define extra variables for the pipeline.

Reply:
Do you mean instead of `LEDGERURI: https://$(ledgerName).confidential-ledger.azure.com` it should say `LEDGERURI: $(ledgerUri)`?

Reply:
Setting these variables in the test-resources.json file should be sufficient, e.g.

```
  "outputs": {
    "LEDGER_NAME": {
      "type": "string",
      "value": "[parameters('ledgerName')]"
    },
    "IDENTITY_SERVICE_URL": {
      "type": "string",
      "value": "[concat('https://identity.confidential-ledger.core.azure.com/ledgerIdentity/', parameters('ledgerName'))]"
    }
  }
```

This has the benefit of the right env variables being set when running the live tests locally, so you don't have to do more manual setup after the deployment.